doc/manual/src/language/index.md

```diff
@@ -1,12 +1,11 @@
 # Nix Language
 
-The Nix language is
+The Nix language is designed for conveniently creating and composing *derivations* – precise descriptions of how contents of existing files are used to derive new files.
```

Comment:
Link glossary?

Reply:
At this point I wouldn't link anything. Also derivations are not that well-documented to encourage people jumping there. Builtins are linked right there.